ui/admin/app/components/form/credential-library/vault-generic/index.js

```diff
@@ -30,4 +30,23 @@ export default class FormCredentialLibraryVaultGenericComponent extends Componen
   get isHttpRequestBodyAllowed() {
     return this.args.model.http_method?.match(/post/i);
   }
+
+  get mappingOverrides() {
+    return options.mapping_overrides[this.args.model.credential_type];
+  }
+
+  /**
+   * Clear the previously selected key value pair when toggling between credential types on a new form
+   *
+   */
+  @action
+  selectCredentialType({ target: { value } }) {
+    if (
```

Comment:
Actually do you need this if check? Couldn't you just set it regardless if they're switching types? It should always be an empty object anyways right?

Reply:
good call, yes, I don't need it, since a user can't update this type once its created, I will remove